examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -13,8 +13,7 @@
    "source": [
```

Comment:
<p>But before, we should define model class&nbsp;<code>'model'</code>&nbsp;and channels positions&nbsp;<code>'channels'</code>&nbsp;(for TensorFlow models -&nbsp;<strong>'last'</strong>, for PyTorch models&nbsp;- <strong>'first'</strong>) in&nbsp;<code>config</code>.</p><p> </p><p>There are also predefined&nbsp;<a href="https://analysiscenter.github.io/batchflow/intro/model_zoo.html" target="_blank">models</a>&nbsp;of both frameworks. You can use them without additional configuration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>